geps/gep-2907/index.md

```diff
@@ -0,0 +1,150 @@
+# GEP-2907: TLS Configuration Placement and Terminology
+
+* Issue: [#2907](https://github.com/kubernetes-sigs/gateway-api/issues/2907)
+* Status: Memorandum
+
+## TLDR
+This GEP aims to define high level TLS terminology and structure within Gateway
+API to ensure that all our independent proposals related to TLS result in a
+coherent set of APIs. This will result in some adjustments to provisional and
+experimental TLS-related GEPs, specifically [BackendTLSPolicy](/geps/gep-1897)
+and [Client Certificate Verification](/geps/gep-91).
+
+## Goals
+* Define high-level terminology for how we refer to TLS in Gateway API.
+* Define top level fields where TLS configuration can live.
+
+## Non-Goals
+* Add or change fields directly. (This may inspire changes in other GEPs
+  though).
+* Commit to including specific parts of TLS configuration in Gateway API. (This
+  is merely to provide space for future configuration, not a commitment that we
+  will add it to the API.)
+
+## Out of Initial Scope
+There are a variety of related TLS concepts in Gateway API that are not currently
+in scope for this GEP. In the future, this GEP may be expanded to include:
+
+1. Automatic mTLS (often associated with Service mesh)
+1. TLS Passthrough
+1. TLSRoute
+
+
+## Introduction
+
+### Where TLS could be configured
+We have three different places where we might want to configure TLS:
+
+#### A. Gateways
+Config attached at this layer will apply to everything behind a single address
+such as a virtual IP.
+
+#### B. Gateway Listeners
+Each Gateway contains multiple “Listeners”. Each HTTPS Listener in a Gateway
+must have a unique combination of protocol, port, and SNI (Server Name
+Indication). TLS configuration attached at this layer should be [isolated via
+SNI](/guides/implementers/#listener-isolation).
+
+#### C. BackendTLSPolicy
+This policy allows us to attach unique TLS configuration per Backend. Depending
+on the organization, this policy may be owned by the application owner or the
+cluster operator. Note that this configuration will be used by all Gateways
+(potentially from different implementations) that are connecting to the backend.
+
+### "Frontend" and "Backend"
+A guiding principle in this naming is to use consistent naming for “Upstream”
+(1+2) and “Downstream” (3+4), similar to Envoy. To avoid the confusion with what
```

Comment:
For `similar to Envoy` terminology, 3+4 are the upstream, 1+2 is downstream. I see what you mean by "avoid confusion" 😃 
```suggestion
A guiding principle in this naming is to use consistent naming for “Upstream”
(3+4) and “Downstream” (1+2), similar to Envoy. To avoid the confusion with what
```

Reply:
🤦  updated, thanks!